openapi/anpr-dashboard.yaml

```diff
@@ -65,7 +65,8 @@ info:
   # Chi posso contattare per informazioni sul servizio e sul suo stato?
   contact:
     email: roberto@teamdigitale.governo.it
```

Comment:
we should have a technical mail group for all these kind of contact.

Reply:
do it please, or set another email :)

Reply:
```suggestion
    email: tech@teamdigitale.governo.it
```